components/terrain/quadtreeworld.cpp

```diff
@@ -341,7 +341,7 @@ void loadRenderingNode(ViewData::Entry& entry, ViewData* vd, int vertexLodMod, f
 
         for (QuadTreeWorld::ChunkManager* m : chunkManagers)
         {
-            if (m->getViewDistance() && entry.mNode->distance(vd->getViewPoint()) > m->getViewDistance())
+            if (m->getViewDistance() && entry.mNode->distance(vd->getViewPoint()) > m->getViewDistance() + reuseDistance*10)
```

Comment:
Probably it would be better to avoid magic numbers. It is unclear why 10 instead of, for example, 8 or 16.

Reply:
I agree but its not so straight forward. The real magic number to calibrate is the preloader loading time and prediction time. Will try to find a better solution in the future. It is not really specific to groundcover.